### DISCUSSION TITLE
Possible to Hide Tags/Billable for end users.

### DISCUSSION THREAD

**Discussion by chetanmadaan** [kimai/kimai#2746] [General]
Hi,

I am trying to create a very simplistic experience for the end user logging in time. Is it possible to disable the tags field and billable checkbox for the user logging in time?

Thanks,
Chetan

**Comment by kevinpapst** [comment]
Use the customcssbundle to write some simple rules for hiding these fields from the form.

**Comment by chetanmadaan** [comment]
Thanks Kevin. but then we have the tags at dashboard too (post login table), I guess that can't be hidden via CSS and would need to edited somewhere in the code right?

**Reply by kevinpapst** [reply]
Post your rule here and we can check if/why they might not work.
